tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/UserResource.java

```diff
@@ -132,6 +137,8 @@ public Response createUsersWithArrayInput(
     )
     public Response createUsersWithListInput(
         @Parameter(
+            name = "createUsersUsingList",
+            in = ParameterIn.QUERY,
```

Comment:
Again - not a query param.  I think all of this is likely leftovers from when this was swagger 2.0.